net/upnp/Makefile

```
@@ -1,7 +1,7 @@
 PLUGIN_NAME=		upnp
-PLUGIN_VERSION=		1.7
+PLUGIN_VERSION=		1.8
 PLUGIN_DEPENDS=		miniupnpd
-PLUGIN_COMMENT=		Universal Plug and Play (UPnP IGD & PCP/NAT-PMP) Service
+PLUGIN_COMMENT=		UPnP IGD & PCP/NAT-PMP Service | Universal Plug and Play
```

"|" is also very uncommon and really not worth the risk introducing

---

how about this? Don't mind that much anymore, just trying to get rid of the fluff

```suggestion
PLUGIN_COMMENT=		UPnP IGD & PCP/NAT-PMP Service
```